pkgs/stdenv/linux/default.nix

```diff
@@ -284,22 +284,13 @@ rec {
       shellPackage = stage4.pkgs.bash;
     };
 
-    /* outputs TODO
-    allowedRequisites = with stage4.pkgs;
-      [ gzip bzip2 xz bash binutils coreutils diffutils findutils gawk
-        glibc gnumake gnused gnutar gnugrep gnupatch patchelf attr acl
-        paxctl zlib pcre linuxHeaders ed gcc gcc.cc libsigsegv
-      ];
-      */
+    overrides = lib.overrideNativeDrvs [
+      "attr" "acl" "bash" "binutils" "bzip2" "coreutils" "diffutils"
+      "findutils" "gawk" "gcc" "glibc" "gnugrep" "gnumake" "gnused"
+      "gnutar" "gnupatch" "gzip" "patchelf" "paxctl" "pcre" "xz"
+      "zlib"
+    ] stage4.pkgs;
```

Comment:
Why is `allowedRequisites` being deleted? IIRC it is some thing to lint the dependencies the derivation has, but I could be wrong.. The commented code is probably horribly bitrotted and probably should be removed with an issue created, but strictly speaking that change his little to do with this PR?